starter-kits/credential-registry/docker/docker-compose.yml

```diff
@@ -259,7 +261,8 @@ services:
       /bin/bash -c "
       echo waiting for solr ...;
       sleep 10;
-      ${STI_SCRIPTS_PATH}/run;"
+      # ${STI_SCRIPTS_PATH}/run;"
```

Comment:
If the `run` script is commented... How does the worker start? 😝 
I think we should be calling the `start-celery-worker` here...

Reply:
Nevermind this, I missed the `APP_SCRIPT` set as an environment variable.